test_runner/src/test/kotlin/ftl/fixtures/test_app_cases/flank-single-parametrized.yml

```diff
@@ -0,0 +1,4 @@
+gcloud:
+  app: ./src/test/kotlin/ftl/fixtures/tmp/apk/app-single-success-parametrized-debug.apk
```

Comment:
Apks shouldn't be added directly to the flank repo, instead we keep them in https://github.com/Flank/test_artifacts to keep the flank repo size small

For the sample app, let's update the source code in this repo so we can easily rebuild the apks. As Android evolves, we'll want to rebuild apks with different options and target platforms.

Reply:
Changed